implementation/src/test/java/io/smallrye/mutiny/operators/MultiOnEventTest.java

```diff
@@ -476,6 +476,22 @@ public void testInvokeUniOnItem() {
         assertThat(res).hasValue(2);
     }
 
+    @Test
+    public void testInvokeUniOnItemWithShortcut() {
+        AtomicInteger res = new AtomicInteger();
+        AtomicInteger twoGotCalled = new AtomicInteger();
+
+        List<Integer> r = numbers.onItem().invokeUni(i -> {
```

Comment:
We should have the same test case, but with `numbers.invokeUni()`

Reply:
Damned, that was the purpose of this test!